swaggerui.html

```diff
@@ -4,9 +4,9 @@
   <meta charset="UTF-8">
   <title>Swagger API</title>
 
-  <link rel="stylesheet" type="text/css" href="//unpkg.com/swagger-ui-dist/swagger-ui.css">
-  <link rel="icon" type="image/png" href="//unpkg.com/swagger-ui-dist/favicon-32x32.png" sizes="32x32"/>
-  <link rel="icon" type="image/png" href="//unpkg.com/swagger-ui-dist/favicon-16x16.png" sizes="16x16"/>
+  <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/npm/swagger-ui.css">
```

Comment:
I think these all need the package name in the URL. https://cdn.jsdelivr.net/npm/swagger-ui.css doesn't resolve but https://cdn.jsdelivr.net/npm/swagger-ui-dist/swagger-ui.css does.

```suggestion
  <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/npm/swagger-ui-dist/swagger-ui.css">
```

Reply:
Yup, missed the package name. Fixed now. Sorry.